src/features/PesterTests.ts

```diff
@@ -51,8 +51,8 @@ export class PesterTestsFeature implements IFeature {
         this.languageClient = languageClient;
     }
 
-    private launchAllTestsInActiveEditor(launchType: LaunchType) {
-        const uriString = vscode.window.activeTextEditor.document.uri.toString();
+    private launchAllTestsInActiveEditor(launchType: LaunchType, fileUri?: vscode.Uri) {
```

Comment:
you made this optional but then use `fileUri.toString()`. Can you either make this not optional or null check the `fileUri` in the method?

```
fileUri ? fileUri.toString() : vscode.window.activeTextEditor.document.uri.toString()
```

Reply:
Thanks for spotting it. This was a left-over from development when I was trying various approaches. Cleaned it up. `fileUri` is not optional any more now